### PR TITLE
Initialize Hello Microverse Project

### PR DESCRIPTION
**Pull request by lucascomamala**
- Created a new branch according to the [GitHub flow] rules
- Setup a linter for HTML and CSS, making sure that any files that are not relevant are ignored by Git
- Added a descriptive Readme
- Added an .html file with the text "Hello Microverse!"
- Added a .css file with styling for the header
- Added a test.md file with the text "This file should be ignored by Git"
- Added a .gitignore file
- Made sure the test.md file is actually ignored by Git and not present in the repository

### CODE REVIEW

**Review by MeqdamAlqudah — COMMENTED** [review]
Hi @lucascomamala , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted the wrong project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+h1 {
+  color: red;
```
Comment (MeqdamAlqudah) [inline style.css]: - [x] Please use a class to style elements, not a general class

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @lucascomamala , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (lucascomamala) [reply style.css]: @MeqdamAlqudah Done!

**Review by mwanawabangona — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @lucascomamala 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@mwanawabangona) in your question so I can receive the notification .

------

**Pull request merged**